brigade/core/inventory.py

```diff
@@ -106,6 +107,25 @@ def __str__(self):
     def __repr__(self):
         return "{}: {}".format(self.__class__.__name__, self.name)
 
+    def get(self, item, default=None):
+        """
+        Returns the value ``item`` from the host or hosts group variables.
+
+        Arguments:
+            item(``str``): The variable to get
+            default(``any``): Return value if item not found
+
+        """
+        try:
```

Comment:
I think this would be cleaner:

```
try:
    return self.__item__(item)
except KeyError:
    return default
```

Basically you are duplicating the code in `self.__item__` so calling it should be enough